src/network-entity.ts

```diff
@@ -131,6 +132,7 @@ export class NetworkEntity implements Initializable {
     const totalCoins = this.publisherSubsidyMinCoins + Math.round(Math.random() * (this.publisherSubsidyMaxCoins - this.publisherSubsidyMinCoins));
     if (!subsidyDay || midnightToday > subsidyDay.starting) {
       console.log("Network.poll: Adding new publisher subsidy day", totalCoins, this.publisherSubsidyCoinsPerOpen);
+      const amount = await cardManager.calculateCurrentPublisherSubsidiesPerPaidOpen(subsidyDay ? subsidyDay.coinsPerPaidOpen : 0.50);
       await db.insertPublisherSubsidyDays(midnightToday, totalCoins, this.publisherSubsidyCoinsPerOpen, this.publisherSubsidyReturnUserBonus);
```

Comment:
Not sure if there's a bug here, but you are not using the 'amount' value anywhere.

Reply:
That was a bug alright.  the "publisherSubsidyCoinsPerOpen" has been replaced with a dynamically calculated "amount" but wasn't using it to populate the right field.  Good catch.